edit-post/assets/stylesheets/_variables.scss

```diff
@@ -56,6 +56,8 @@ $block-parent-side-ui-clearance: $parent-block-padding - $block-padding; // spac
 
 $block-container-side-padding: $block-side-ui-width + $block-padding + 2 * $block-side-ui-clearance;
 
+$default-block-margin: $block-padding * 2 + $block-spacing; // This is the default margin between blocks.
```

Comment:
When you hover or select a block, we paint the outline _outside_ the footprint of the block. 

This means that if a block has no margin, these outlines will _overlap_. This variable is designed in a way so as to prevent this overlap from happening.